lib/service/service.go

```diff
@@ -355,6 +356,13 @@ func (process *TeleportProcess) addConnector(connector *Connector) {
 	process.connectors[connector.ClientIdentity.ID.Role] = connector
 }
 
+func (process *TeleportProcess) setAuthServerFeatures(features proto.Features) {
```

Comment:
You also need a counterpart getter that returns features under process.Lock() and use that instead of accessing them directly.

Reply:
defined a getter